otx/algorithms/action/configs/classification/movinet/template.yaml

```diff
@@ -0,0 +1,63 @@
+# Description.
+model_template_id: Custom_Action_Classificaiton_MoViNet
+name: MoViNet
+task_type: ACTION_CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Basic transfer learning template for MoViNet
+application: ~
+
+# Algo backend.
+framework: OTXAction v2.9.1
+
+# Task implementations.
+entrypoints:
+  base: otx.algorithms.action.tasks.ActionTrainTask
+  openvino: otx.algorithms.action.tasks.ActionOpenVINOTask
+
+# Capabilities.
+capabilities:
+  - compute_representations
+
+# Hyperparameters.
+hyper_parameters:
+  base_path: ../configuration.yaml
+  parameter_overrides:
+    learning_parameters:
+      batch_size:
+        default_value: 8
+        auto_hpo_state: POSSIBLE
+      learning_rate:
+        default_value: 0.003
```

Comment:
Is this come from experiments? I think it is quite large

Reply:
Performance improvement will be dealt in another PR. Thank you for your comment :)